### PR TITLE
Colorbar

### PR DESCRIPTION
**Pull request by felipeaoli**
Fixes after refactoring and changes in colorbars and colormaps.

### CODE REVIEW

File: backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py [hunk 1]
```diff
@@ -116,18 +116,42 @@ def circle(
             hover_color=hover_color,
             hover_alpha=hover_alpha,
             hover_line_color=hover_line_color,
-            radius=radius,
+            #radius=radius,
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py]: If this is not going to be used please remove all occurrences of radios on `.circle`

File: backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py [hunk 2]
```diff
@@ -116,18 +116,42 @@ def circle(
...
+            ztext = ['{:6.2f}'.format(i) for i in dz_valid]
+            cbarformat = "%5.2f"
+        return ztext, cbarformat
+
     def colorbar(self, mapper, title='counts'):
+        formatter = PrintfTickFormatter(format="%4.2f")
+
         color_bar = ColorBar(color_mapper=mapper, label_standoff=16,
-                                title=title,
+                                title= "(Val-Ref)", #title,
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py]: If this is going to be the default value for all colorbars please remove `colorbar_title`

File: backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py (hunk 1 above)
Comment (felipeaoli) [reply backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py]: Done.

File: backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py (hunk 2 above)
Comment (felipeaoli) [reply backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py]: Sure, now I just changed the default value

**Pull request merged**